SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialActorUtils.h

```diff
@@ -2,18 +2,43 @@
 
 #pragma once
 
-#include "CoreMinimal.h"
+#include "EngineClasses/SpatialNetConnection.h"
 
 #include "Components/SceneComponent.h"
+#include "Containers/Array.h"
+#include "Containers/UnrealString.h"
 #include "Engine/EngineTypes.h"
-#include "EngineClasses/SpatialNetConnection.h"
 #include "GameFramework/Actor.h"
 #include "GameFramework/Controller.h"
 #include "GameFramework/PlayerController.h"
+#include "Math/Vector.h"
 
 namespace SpatialGDK
 {
 
+// We need the optional IgnoredActor argument because Actor deletions are broadcast before
+// SetOwner is called and ownership hierarchies are updated.
+inline TArray<AActor*> GetOwnershipHierarchyPath(const AActor* Actor, const AActor* IgnoredActor = nullptr)
+{
+	check(Actor != nullptr);
+
+	AActor* OwnerIterator = Actor->GetOwner();
+	if (OwnerIterator == nullptr)
+	{
+		return TArray<AActor*>();
+	}
+
+	TArray<AActor*> OwnershipHierarchy {OwnerIterator};
```

Comment:
This line is slightly confusing as the iterator is also the element and this is using an intialiser list constructor. I think just making it `= {OwnerIterator}` might make it better. However It might be that just calling it `Owner` would be clearer.

Reply:
Also my preference for constructing something with an initializer list is to use =.